<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QWidget" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QImageViewer - Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/res/settings.png</normaloff>:/res/settings.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabMainSettings">
      <attribute name="title">
       <string>Main settings</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="locale">
            <locale language="English" country="UnitedStates"/>
           </property>
           <property name="text">
            <string>Default folder:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="defaultfolderLineEdit"/>
         </item>
         <item>
          <widget class="QPushButton" name="defaultfolderBrowseButton">
           <property name="locale">
            <locale language="English" country="Kenya"/>
           </property>
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="mouseFullscreenCheckBox">
         <property name="text">
          <string>Double-click enables/disables full screen mode</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="mouseZoomCheckBox">
         <property name="text">
          <string>Enable/Disable mouse wheel zoom</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Slideshow interval:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="slideshowIntervalSpinBox">
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="minimum">
            <double>0.100000000000000</double>
           </property>
           <property name="maximum">
            <double>60.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="slideshowIntervalButton">
           <property name="text">
            <string>Reset</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>60</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QCheckBox" name="slideshowTransitionCheckBox">
           <property name="text">
            <string>Smooth transition in slideshow</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Application language:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="languageComboBox">
           <item>
            <property name="text">
             <string>System</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>English</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Русский</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Українська</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Polski</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Design</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Fullscreen background color</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="fullscreenColorLabel">
           <property name="minimumSize">
            <size>
             <width>27</width>
             <height>27</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>27</width>
             <height>27</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="fullscreenColorButton">
           <property name="text">
            <string>Choose</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Icon Pack:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="iconpackBox">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="currentIconLabel">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>17</width>
           <height>268</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Panel</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QComboBox" name="toolpanelComboBox">
           <property name="minimumSize">
            <size>
             <width>275</width>
             <height>0</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>Panel at the bottom</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Panel on top</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>No panel</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>166</width>
            <height>298</height>
           </rect>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
           </property>
           <item row="0" column="0">
            <widget class="QCheckBox" name="panelbutRotateLeftCheckBox">
             <property name="text">
              <string>Rotate Left</string>
             </property>
             <property name="icon">
              <iconset resource="../resources.qrc">
               <normaloff>:/res/rotate-left.png</normaloff>:/res/rotate-left.png</iconset>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QCheckBox" name="panelbutRotateRightCheckBox">
             <property name="text">
              <string>Rotate Right</string>
             </property>
             <property name="icon">
              <iconset resource="../resources.qrc">
               <normaloff>:/res/rotate-right.png</normaloff>:/res/rotate-right.png</iconset>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QCheckBox" name="panelbutFlipHorizontalCheckBox">
             <property name="text">
              <string>Flip hotizontal</string>
             </property>
             <property name="icon">
              <iconset resource="../resources.qrc">
               <normaloff>:/res/flip-horizontal.png</normaloff>:/res/flip-horizontal.png</iconset>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QCheckBox" name="panelbutFlipVerticalCheckBox">
             <property name="text">
              <string>Flip Vertical</string>
             </property>
             <property name="icon">
              <iconset resource="../resources.qrc">
               <normaloff>:/res/flip-vertical.png</normaloff>:/res/flip-vertical.png</iconset>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QCheckBox" name="panelbutZoomInCheckBox">
             <property name="text">
              <string>Zoom In</string>
             </property>
             <property name="icon">
              <iconset resource="../resources.qrc">
               <normaloff>:/res/zoom-in.png</normaloff>:/res/zoom-in.png</iconset>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QCheckBox" name="panelbutZoomOutCheckBox">
             <property name="text">
              <string>Zoom Out</string>
             </property>
             <property name="icon">
              <iconset resource="../resources.qrc">
               <normaloff>:/res/zoom-out.png</normaloff>:/res/zoom-out.png</iconset>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QCheckBox" name="panelbutZoomWindowCheckBox">
             <property name="text">
              <string>Zoom to window size</string>
             </property>
             <property name="icon">
              <iconset resource="../resources.qrc">
               <normaloff>:/res/zoom-window.png</normaloff>:/res/zoom-window.png</iconset>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QCheckBox" name="panelbutZoomOriginalCheckBox">
             <property name="text">
              <string>Zoom to original size</string>
             </property>
             <property name="icon">
              <iconset resource="../resources.qrc">
               <normaloff>:/res/zoom-original.png</normaloff>:/res/zoom-original.png</iconset>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QCheckBox" name="panelbutFullscreenCheckBox">
             <property name="text">
              <string>Fullscreen</string>
             </property>
             <property name="icon">
              <iconset resource="../resources.qrc">
               <normaloff>:/res/fullscreen.png</normaloff>:/res/fullscreen.png</iconset>
             </property>
            </widget>
           </item>
           <item row="9" column="0">
            <widget class="QCheckBox" name="panelbutSlideshowCheckBox">
             <property name="text">
              <string>Slideshow</string>
             </property>
             <property name="icon">
              <iconset resource="../resources.qrc">
               <normaloff>:/res/slideshow.png</normaloff>:/res/slideshow.png</iconset>
             </property>
            </widget>
           </item>
           <item row="10" column="0">
            <widget class="QCheckBox" name="panelbutPropertiesCheckBox">
             <property name="text">
              <string>Properties</string>
             </property>
             <property name="icon">
              <iconset resource="../resources.qrc">
               <normaloff>:/res/file-properties.png</normaloff>:/res/file-properties.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QPushButton" name="panelReset">
           <property name="text">
            <string>Reset panel settings</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Hotkeys</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTreeWidget" name="hotkeyWidget">
         <property name="columnCount">
          <number>2</number>
         </property>
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string notr="true">2</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QPushButton" name="resetHotkeysButton">
           <property name="text">
            <string>Reset hotkeys to defaults</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_8">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="resetButton">
       <property name="text">
        <string>Reset all settings</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="acceptButton">
       <property name="text">
        <string>Accept new settings</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
